docs/guides/connecting_to_your_data/filesystem/pandas.md

```diff
@@ -103,14 +103,37 @@ Save the configuration into your `DataContext` by using the `add_datasource()` f
 
 Verify your new Datasource by loading data from it into a `Validator` using a `BatchRequest`.
 
-Add the path to your CSV in the `path` key under `runtime_parameters`.
+<Tabs
+  defaultValue='runtime_batch_request'
+  values={[
+  {label: 'Specify a path to single CSV', value:'runtime_batch_request'},
+  {label: 'Specify a data_asset_name', value:'batch_request'},
+  ]}>
+  <TabItem value="runtime_batch_request">
+
+Add the path to your CSV in the `path` key under `runtime_parameters` in your `BatchRequest`.
 
-```python file=../../../../integration/code/connecting_to_your_data/filesystem/pandas_yaml_example.py#L39-L45
+```python file=../../../../integration/code/connecting_to_your_data/filesystem/pandas_yaml_example.py#L40-L46
 ```
 Then load data into the `Validator`.
-```python file=../../../../integration/code/connecting_to_your_data/filesystem/pandas_yaml_example.py#L53-L59
+```python file=../../../../integration/code/connecting_to_your_data/filesystem/pandas_yaml_example.py#L53-L60
 ```
 
+  </TabItem>
+  <TabItem value="batch_request">
+
+Add the name of the data asset to the `data_asset_name` in your `BatchRequest`.
+
```

Comment:
One non-blocking suggestion here. Could we add an additional sentence that is something like: "the data_asset_name can be found in the output of `test_yaml_config()` ?

Reply:
Good call - added.